### PR TITLE
[2.5] Update provider detector to include rke.windows

### PR DESCRIPTION
**Pull request by StrongMonkey**
Backport of https://github.com/rancher/rancher/pull/29691

### CODE REVIEW

**Comment by StrongMonkey** [comment]
https://github.com/rancher/rancher/issues/28947

File: go.mod [hunk 1]
```diff
@@ -83,7 +83,7 @@ require (
 	github.com/rancher/apiserver v0.0.0-20201023000256-1a0a904f9197
 	github.com/rancher/dynamiclistener v0.2.1-0.20200910203214-85f32491cb09
 	github.com/rancher/eks-operator v1.0.4
-	github.com/rancher/kubernetes-provider-detector v0.1.0
+	github.com/rancher/kubernetes-provider-detector v0.1.1-0.20201022034134-8f971d9f77cf
```
Comment (dramich) [inline go.mod:86]: I think we should have a new release

Comment (dramich) [reply go.mod:86]: https://github.com/rancher/kubernetes-provider-detector/releases/tag/v0.1.1

Comment (StrongMonkey) [reply go.mod]: @dramich Updated.

**Review by dramich — APPROVED** [review]

**Pull request merged**